src/java/edu/ucsd/library/dams/api/FedoraAPIServlet.java

```diff
@@ -898,20 +898,9 @@ private void outputTransform( String objid, String cmpid, String fileid,
 				Document doc = DocumentHelper.parseText(rdfxml);
 				String discoverGroup = accessGroup( doc, true, objid );
 				String accessGroup = accessGroup( doc, false, objid );
-				String adminGroup = doc.valueOf(
-					"//dams:unitGroup"
-				);
-				if ( adminGroup != null && !adminGroup.equals("") )
-				{
-					// use linked admin group if available
-					params.put("adminGroup", new String[]{adminGroup});
-				}
-				else
-				{
-					// use default admin group2 if none specified
-					params.put("adminGroup", new String[]{roleAdmin});
-					params.put("adminGroup2", new String[]{roleAdmin2});
-				}
+				String adminGroup = roleAdmin;
+				params.put("adminGroup", new String[]{adminGroup});
+				params.put("adminGroup2", new String[]{roleAdmin2});
```

Comment:
It's configured in dams.properties: https://lib-stash.ucsd.edu/projects/ND/repos/private_config/browse/damsrepo/production/dams.properties#58 .